crates/sui-single-node-benchmark/src/main.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use clap::Parser;
+use sui_single_node_benchmark::command::Command;
+use sui_single_node_benchmark::execution::{
+    benchmark_move_transactions, benchmark_simple_transfer,
+};
+
+#[tokio::main]
+async fn main() {
+    let _guard = telemetry_subscribers::TelemetryConfig::new()
+        .with_log_level("off,sui_single_node_benchmark=info")
```

Comment:
Does this override `RUST_LOG` env var?

Reply:
No, the next line (with_env) will override log level if it contains RUST_LOG